### PR TITLE
Refactoring ldapgroupEdit page and correcting some spelling englisch …

### PR DESCRIPTION
**Pull request by IkramMaalej**
…mistakes.

### CODE REVIEW

**Comment by henning-gerhardt** [comment]
There are merge conflicts in files `Kitodo/src/main/webapp/WEB-INF/templates/includes/header.xhtml`, `Kitodo/src/main/webapp/pages/ldapgroupEdit.xhtml` and `Kitodo/src/main/webapp/pages/searchProcess.xhtml`. Please solve this conflicts.

File: Kitodo/src/main/resources/messages/messages_en.properties [hunk 1]
```diff
@@ -186,7 +187,13 @@ benutzergruppen=User groups
 benutzergruppenHinzufuegen=Add user groups
 benutzerkonfiguration=User configuration
 benutzerprofilLaden=Load user profile
+<<<<<<< HEAD
```
Comment (Beacze) [inline Kitodo/src/main/resources/messages/messages_en.properties]: Necessary to fix rebase.

File: Kitodo/src/main/resources/messages/messages_en.properties [hunk 2]
```diff
@@ -186,7 +187,9 @@ benutzergruppen=User groups
 benutzergruppenHinzufuegen=Add user groups
 benutzerkonfiguration=User configuration
 benutzerprofilLaden=Load user profile
-benutzerverwaltung=User management
+benutzerverwaltung=Users management
+benutztShell=Uses shell
```
Comment (henning-gerhardt) [inline Kitodo/src/main/resources/messages/messages_en.properties]: This "new" translation is not used and was maybe removed by Beatrycze in a precious pull request. So do not add it again.

File: Kitodo/src/main/resources/messages/messages_en.properties [hunk 3]
```diff
@@ -186,7 +187,9 @@ benutzergruppen=User groups
...
+berechnungen=Calculations
```
Comment (henning-gerhardt) [inline Kitodo/src/main/resources/messages/messages_en.properties]: This "new" translation is not used and was maybe removed by Beatrycze in a precious pull request. So do not add it again.

File: Kitodo/src/main/webapp/pages/ldapgroupEdit.xhtml [hunk 4]
```diff
@@ -42,9 +42,10 @@
                               icon="fa fa-times fa-lg" iconPos="right" styleClass="secondary-button"/>
                 </p:panel>
                 <!-- main form -->
-                <p:tabView id="ldabgroupTabView">
+                <p:messages id="useredit-messages" autoUpdate="true" /> 
+		<p:tabView id="ldabgroupTabView">
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/pages/ldapgroupEdit.xhtml]: We are using space indent, you are replacing it with tab indent.

File: Kitodo/src/main/webapp/pages/ldapgroupEdit.xhtml [hunk 5]
```diff
@@ -42,9 +42,10 @@
...
                     <p:tab id="labgroupTab" title="#{msgs.details}">
-                        <p:panelGrid columns="2" layout="grid">
+			<p:panelGrid columns="2" layout="grid">
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/pages/ldapgroupEdit.xhtml]: Same here: you are replacing space indent with tab indent.

File: Kitodo/src/main/webapp/pages/ldapgroupEdit.xhtml [hunk 6]
```diff
@@ -182,21 +183,40 @@
...
                             </p:row>
-                            <p:commandButton value="#{msgs.loeschen}"
-                                             onclick="return confirm('#{msgs.sollDieserEintragWirklichGeloeschtWerden}')"
-                                             action="#{LdapGruppenForm.Loeschen}"
-                                             rendered="#{LdapGruppenForm.myLdapGruppe.id ne null}"
-                                             icon="fa fa-trash-o fa-lg"
-                                             iconPos="right"/>
                         </p:panelGrid>
-                    </p:tab>
+			<p:panel styleClass="content-header">
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/pages/ldapgroupEdit.xhtml]: This whole block is indent by tab but we are using space indent.

**Pull request merged**